app/edge/devices/components/AMTDevicesDatatable/columns/status.tsx

```diff
@@ -0,0 +1,22 @@
+import {CellProps, Column, TableInstance} from 'react-table';
```

Comment:
Spaces for consistency?  `import { CellProps, Column, TableInstance } from 'react-table';`